ansible/roles/linux-common/tasks/main.yml

```diff
@@ -43,3 +43,9 @@
 - name: Clean up after ourselves
   ansible.builtin.apt:
     clean: yes
+
+- name: Install ansible-init playbook
```

Comment:
Lets try get the drivers installed in the image, but then load the module at boot time.